server/datastore/mysql/schema.sql

```diff
@@ -76,7 +76,7 @@ CREATE TABLE `cve_meta` (
   `cvss_score` double DEFAULT NULL,
   `epss_probability` double DEFAULT NULL,
   `cisa_known_exploit` tinyint(1) DEFAULT NULL,
-  `published` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `published` timestamp NULL DEFAULT NULL,
```

Comment:
The auto-generated SQL schema had changes for me, I _think_ this is something @michalnicp added recently, does that look correct to you?

Reply:
This looks correct according to the migration in `server/datastore/mysql/migrations/tables/20220526123327_RenameCVEScoresToCVEMeta.go`. I may have forgotten to commit the changes to `schema.sql` on one of my recent prs.